publish.sh

```diff
@@ -7,7 +7,7 @@ if [[ $test_output == *"failing"* ]]; then
     echo "$test_output"
 else
     vsce package
-    sed -e 's/"publisher": "kreativ-software"/"publisher": "jsw"/g' ./package.json >./package.json.temp
+    sed -e 's/"publisher": "tdallau-csharpextensions"/"publisher": "jsw"/g' ./package.json >./package.json.temp
```

Comment:
Why did you change the publisher here? I don't see any packages in the vscode extensions that you authored:
![image](https://user-images.githubusercontent.com/20839414/119120092-2b236780-ba2c-11eb-9114-6c6885143b19.png)

Reply:
He made [that extension](https://marketplace.visualstudio.com/items?itemName=tdallau-csharpextensions.csharpextensions). Check the link url his nickname is there.

Reply:
Thnx for pointing out my error! 😊 I have been to flimsy in my research it seems.